src/redux/books/books.js

```diff
@@ -1,27 +1,42 @@
-const ADD_BOOK = 'redux/books/book/ADD_BOOK';
-const REMOVE_BOOK = 'redux/books/book/REMOVE_BOOK';
+import { createSlice } from '@reduxjs/toolkit';
+import { v4 as uuidv4 } from 'uuid';
 
-const initialState = [];
-
-const newBook = (book) => ({
-  type: 'ADD_BOOK',
-  book,
-});
-
-const removeBook = (id) => ({
-  type: 'REMOVE_BOOK',
-  id,
+const bookSlice = createSlice({
+  name: 'books',
+  initialState: [
+    {
+      id: uuidv4(),
+      title: 'The Smart parent',
+      aythor: 'Nancy Van Pelt',
+    },
+    {
+      id: uuidv4(),
+      title: 'Choose Your Hard',
+      aythor: 'Recillah Khamala',
+    },
+    {
+      id: uuidv4(),
+      title: 'The hunger games',
+      author: 'Suzanne Collins',
+    },
+    {
+      id: uuidv4(),
+      title: 'Dune',
+      author: 'Frank Herbert',
+    },
+    {
+      id: uuidv4(),
+      title: 'Capital in the Twenty-First Century',
+      author: 'Suzanne Collins',
+    },
+  ],
+  reducers: {
+    newBook: (initialState, action) => [...initialState, action.payload],
```

Comment:
- [ ] Please add `state` instead of `intialState` here in newBook action creator line 34 because you add the new book data to the current `state` itself, not the `initalState`.